tools/osquery/in-a-box/docker-compose.yml

```diff
@@ -59,6 +59,7 @@ services:
       # This can be configured for testing purposes but otherwise uses the
       # typical default of provided.
       FLEET_OSQUERY_HOST_IDENTIFIER: ${FLEET_OSQUERY_HOST_IDENTIFIER:-provided}
+      FLEET_DEV_MAINTAINED_APPS_BASE_URL: https://raw.githubusercontent.com/fleetdm/fleet/refs/heads/add-amazon-dcv/ee/maintained-apps/outputs
```

Comment:
@lucasmrod @iansltx does this approach look right? 

I'm trying to run through [these steps](https://github.com/fleetdm/fleet/pull/28831/files#diff-be469dd148f0c50ad56489c48bdb514522e1a46d21336e8f747b5880d71a6d1bR78) after Lucas pointed out how to specify the env variable we need for fleetctl preview [here](https://github.com/fleetdm/fleet/pull/28441#discussion_r2073664772).

Reply:
It seems you are missing setting the `--preview-config-path ./tools/osquery/in-a-box` flag in `fleetctl preview` (see [here](https://github.com/fleetdm/fleet/pull/28441#discussion_r2073664772)) after editing the  `./tools/osquery/in-a-box/docker-compose.yml`.

Reply:
@noahtalerman Only thing that you might be missing is running `fleetctl trigger --name=maintained_apps`. You wouldn't normally need to do this if the system was being built from scratch (FMA ingest would happen on server start the first time) but I'm not sure what state `fleetctl preview` puts things in.

I just validated that DCV shows up after running the cron when using the most recent `fleet` server that I compiled (alongside the normal `docker-compose` setup):

<img width="898" alt="image" src="https://github.com/user-attachments/assets/ea556b4e-5cfc-4759-b52c-5f1c44a55269" />